tests/kuttl/tests/keystone_resources/02-assert.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Check for:
+#
+# - 1 KeystoneAPI CR
+# - 1 KeystoneService CR
+# - 1 KeystoneEndpoint CR
+#
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneAPI
+metadata:
+  finalizers:
+  - KeystoneAPI
+  - KeystoneService-placement
+  - KeystoneEndpoint-placement
+  name: keystone
+---
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneService
+metadata:
+  finalizers:
+  - KeystoneService
```

Comment:
We have to add finalizer from KeystoneEndpint to KeystoneService because endpoint requires the service. I'm fixing it by https://github.com/openstack-k8s-operators/keystone-operator/pull/265 .

Reply:
that PR was merged so I updated this.